package.json

```diff
@@ -37,5 +37,8 @@
   },
   "resolutions": {
     "usehooks-ts@^2.7.2": "patch:usehooks-ts@npm:^2.7.2#./.yarn/patches/usehooks-ts-npm-2.7.2-fceffe0e43.patch"
+  },
+  "dependencies": {
+    "update-browserslist-db": "latest"
```

Comment:
Do we need this package? also it should be added inside `pacakges/nextjs/pacakge.json` 

To remove this run : `yarn remove update-browserslist-db`

To add it you could first `cd packages/nextjs` and then `yarn add update-browserslist-db`

Reply:
![screenshot](https://github.com/BuidlGuidl/batch1.buidlguidl.com/assets/93211672/11676a8b-abdc-4fc6-8581-bd988390cc00)

I am actually not very sure if this package is needed. I get this notification almost every time I run `yarn start`  . Just to keep this updated, I will move it to my nextJS package. Thank you for the advice!